lib/pino-transport.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const { Writable } = require('stream')
+const { MongoClient } = require('mongodb')
+const parseMongoUrl = require('muri')
```

Comment:
I think you need to remove `muri` as a dependency as it block the usage for `MongoDB Atlas` which have a protocol `mongodb+srv`.

Related Issue: #14

As `client.db()` will use the database name in the `uri`, we can use pass `database` option inside.